decidim-core/app/assets/stylesheets/decidim/application.scss

```diff
@@ -1,3 +1,2 @@
 @import "decidim";
-@import "extras/*";
```

Comment:
Yes, there was just one file in extras and I removed it so it's no longer required.